src/App.jsx

```diff
@@ -1,15 +1,33 @@
-import { useState } from 'react'
-import Header from './Components/Header'
-
+import { useState } from "react";
+import { Routes, Route } from "react-router-dom";
+import "./App.css";
+import Header from "./Components/Header";
+import Home from "./Components/Home";
+import NavBar from "./Components/NavBar";
+import Topics from "./Components/Topics";
+import Article from "./Components/Article";
+import Articles from "./Components/Articles";
+import PostArticle from "./Components/PostArticle";
+import Users from "./Components/Users";
 
 function App() {
-  const [count, setCount] = useState(0)
-
-  return (
-   <>
-    <Header />
-   </>
-  )
+	return (
+		<>
+			<Header />
+			<NavBar />
+			<Routes>
+				<Route path="/" element={<Home />}></Route>
+				<Route path="/topics" element={<Topics />}></Route>
+				<Route path="/articles" element={<Articles />}></Route>
+				<Route path="/articles/:article_id" element={<Article />}></Route>
+				<Route
```

Comment:
Try to make sure each branch you create only has the relevant code for that particular task - e.g. we don't need the post articles or individual articles routes or components just yet